src/Router.tsx

```diff
@@ -1,13 +1,18 @@
 import { BrowserRouter, Route, Routes } from "react-router-dom";
 import { GiftPage } from "./pages/GiftPage";
 import { RedeemPage } from "./pages/RedeemPage";
+import { TopUpWalletAddressPage } from "./pages/TopUpPage";
+import { TopUpARPage } from "./pages/ArTopUpPage";
 
 export function Router() {
   return (
     <BrowserRouter basename={`${import.meta.env.BASE_URL}`}>
       <Routes>
         <Route path="/gift" element={<GiftPage />} />
         <Route path="/redeem" element={<RedeemPage />} />
+        <Route path="/top-up" element={<TopUpWalletAddressPage />} />
+        <Route path="/ar-top-up" element={<TopUpARPage />} />
```

Comment:
nit: /top-up/arweave /top-up/fiat. with top-up linking to both pages?

Reply:
/top-up/crypto with a dropdown on the page of what cyrpto component you want to top up with